src/tests/SearchForm.test.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
This file should be in the `src/tests/components/` directory where all the other tests are